tests/s3/multipart_upload/test_multipart_getput.py

```diff
@@ -829,3 +829,72 @@ def test_multipart_upload_test_40256(self):
         s3_background_io.stop()
         s3_background_io.cleanup()
         self.log.info("ENDED: Test Simple and Multipart upload of an object")
+
+    @pytest.mark.tags('TEST-40745')
+    @pytest.mark.s3_ops
+    @CTFailOn(error_handler)
+    def test_multipart_upload_test_40745(self):
+        """
+        This test is for parallel upload of different objects via simple and multipart upload
+        """
+        mp_config = MPART_CFG["test_28539"]
+        self.log.info("STARTED: test head-object of multipart uploaded object")
+        uploaded_parts, keys, s3_background_io = self.s3_mpu_test_obj.start_ios_get_precalc_parts(
+            mp_config, self.mp_obj_path, log_prefix="TEST-40745_s3bench_ios", duration="0h3m",
+            s3_test_lib_obj=self.s3_test_obj)
+        random.shuffle(keys)
+        process_mpu = multiprocessing.Process(target=self.initiate_multipart,
+                                args=(self.bucket_name, self.object_name),
+                                kwargs={"parts": uploaded_parts,
+                                        "is_part_upload":True, "is_lst_complete_mpu":True})
+        process_put = multiprocessing.Process(target=self.s3_test_obj.put_object,
+                                              args=(self.bucket_name, self.object_name+str("put"),
```

Comment:
Let's store the name of the object being uploaded in line 851 in a variable and use that across the test instead

Eg. object_name_put = self.object_name + "_put"

Reply:
aggreed